packages/core/src/components/forms/_control-group.scss

```diff
@@ -215,6 +215,35 @@ Styleguide components.forms.control-group
   }
 
   /*
+  Responsive control groups
+
+  Add the class `pt-fill` to a control group to make all controls expand equally to fill the
+  available space. Then add the class `pt-fixed` to individual controls to revert them to their
+  original default sizes.
```

Comment:
Let's stick with `pt-fixed` for now, just for consistency...and because I can't think of a better name.